packages/editor/src/panels/hierarchy/hierarchynode.tsx

```diff
@@ -394,7 +395,7 @@ export default React.memo(function HierarchyTreeNode(props: ListChildComponentPr
         !visible ? 'text-text-inactive' : '',
         selected ? 'rounded-sm border border-ui-select-outline bg-ui-select-background text-text-primary' : '',
         isOverOn && canDropOn ? 'border border-dotted' : '',
-        hideGlbChildrenFeatureFlag[0] && isOverOn && !canDropOn ? 'border border-dotted bg-ui-hover-error' : ''
+        !showGlbChildrenFeatureFlag && isOverOn && !canDropOn ? 'border border-dotted bg-ui-hover-error' : ''
```

Comment:
this needs to still access the 0th index or to deconstruct the returned value, since useFeatureFlags returns an array